tox.ini

```diff
@@ -265,7 +265,7 @@ commands =
     sphinx-build                           \
         -b html -d "{envtmpdir}/doctrees"  \
         "{toxinidir}/docs"                 \
-        "{toxworkdir}/docs/html"
```

Comment:
This is to match other projects, and similar to `/htmlcov/`.

Reply:
Just curious, other hyper projects, or which other projects?

Reply:
Klein in particular; I'm trying to keep the configs relatively similar for sanity.

Reply:
/me whispers "we need a tool to keep them in sync"

Reply:
I KNOW